ssm-automation/README.md

```diff
@@ -80,10 +146,11 @@ To create a PAA or PAI with multiple approvers follow these steps:
 5. Begin creating a new Template.
 6. For the Runbook option, choose the Automation Document you created.
 7. Choose the desired approvers and fill in the rest of the fields.
-8. Submit the Template for review. **Note:** Make sure that you have a "Template Reviewer" registered in the "Settings" tab of Change Manager.
-9. Approve the Template.
+8. Submit the Template for review. **Note:** Make sure that you have the Template Approver Role (described in the "Template Approver Role" section above) registered in the "Settings" tab of Change Manager under "Template Reviewer".
```

Comment:
You can easily create links to headers, see https://www.markdownguide.org/extended-syntax/#heading-ids